socorro/lib/datetimeutil.py

```diff
@@ -143,6 +143,12 @@ def uuid_to_date(uuid, century='20'):
 
 
 class JsonDTEncoder(json.JSONEncoder):
+    """JSON encoder that handles datetimes
+
+    >>> json.dumps(some_data, cls=JsonDTEncoder)
+    ...
+
+    """
```

Comment:
Figured I'd add a docstring to this since I use it periodically and always have to look it up.